file_paths.md

```diff
@@ -339,6 +340,39 @@ The package will install a service named `puppetmaster`, create a
         puppetmaster *                    # writeable by puppetmaster
             puppetmaster.pid
 
+# pxp-agent
+
+This package executes Puppet and replies to PXP requests.
```

Comment:
Could you add the name of the pxp-agent service on *nix and windows, e.g. `pxp-agent` or `pxp-agentd` to the section we talk about other services?